solutions/usaco-940.mdx

```diff
@@ -87,6 +87,62 @@ public class Factory {
 ```
 
 </JavaSection>
+<CPPSection>
+
+```cpp
+#include<bits/stdc++.h>
+using namespace std;
+
+signed main(){
+    // uncomment before submitting to usaco
+    // freopen("factory.in", "r", stdin);
+    // freopen("factory.out", "w", stdout);
```

Comment:
sus

Reply:
please explain more, should I remove that?

Reply:
yep! the code should work when submitted to the usaco website